lib/stringify.js

```diff
@@ -87,9 +92,9 @@ var stringify = function stringify( // eslint-disable-line func-name-matching
 
     var objKeys;
     if (Array.isArray(filter)) {
-        objKeys = filter;
+        objKeys = filter.map(mapKeys);
```

Comment:
this will cause three arguments to be passed to `mapKeys`, perhaps we want `function (k) { return mapKeys(k); }` to limit the observable API?